### DISCUSSION TITLE
-H 'Host: aa.work.com' do not work

### DISCUSSION THREAD

**Discussion by yaochangfei** [projectdiscovery/nuclei#693] [Q&A] [answered]
when i use nuclei , i want to set header Host ,so I  use the command :
nuclei -H 'Host: aa.work.com' -t xxx -u xxx -json -irr -o test.json -debug
but  from the debug log ,i see the request Header Host is not change to 'Host: aa.work.com',
where i use -H 'A:2' ,it works 
why? how to set header Host?

**Comment by ehsandeep** [comment, marked as answer]
@yaochangfei You can do this within the template with `unsafe: true`, more details [here](https://nuclei.projectdiscovery.io/templating-guide/protocols/http/#unsafe-http-requests).

**Reply by ehsandeep** [reply]
@yaochangfei this is now doable with the latest release of nuclei https://github.com/projectdiscovery/nuclei/releases/tag/v2.3.6
